### ISSUE TITLE
[1.7.10] Dense Ores problem

### ISSUE THREAD

**Issue by MrHellsing** [CyanideX/Unity#138]
So... After a year of ~~doing nothing~~ <sub><sup>understanding how GitHub works, <sub><sup>[insert_your_variant_here]</sup></sub></sup></sub>
I've actually decided to post here... 
[The problem itself](http://imgur.com/a/CHbws)
And as I've searched through PRs, #24 caused this

**Comment by InsomniaKitten** [comment]
What is the problem? It looks normal to me

**Comment by MrHellsing** [comment]
Somewhat hard to differentiate from the first glance

**Comment by CyanideX** [comment]
So you're saying that you wouldn't mine diamond unless it was dense?

**Comment by MrHellsing** [comment]
No, I'm just saying what they look a bit too similar

**Comment by InsomniaKitten** [comment]
Next time please explain your issue, you merely said "problem" and didn't explain what the problem was

_Sent from my Oneplus A0001 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Managarmr719** [comment]
I agree with @MrHellsing, while it's not that much of an issue, I would still like them to be a bit easier to differentiate.

**Comment by redawgts** [comment]
I also agree with @MrHellsing and @Managarmr719.
